doc/source/conf.py

```diff
@@ -388,7 +388,7 @@
 # html_split_index = False
 
 # If true, links to the reST sources are added to the pages.
-# html_show_sourcelink = True
+# html_show_sourcelink = False
```

Comment:
Even I had a similar doubt.
But since someone in the comments had suggested it so I implemented it.

Reply:
I guess you need to uncomment this line besides changing thr value.

But you should build the docs locally and check that whatever you changed works as expected.

Reply:
I have uncommented that line
Since the changes are reflected on the API pages I have no idea on how to check if it is working as expected. 
If you could just guide me on that it will be very handy since I'm new to this open source contribution field